maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -21,18 +20,38 @@ class DatasetCatalog(object):
             "coco/val2014",
             "coco/annotations/instances_valminusminival2014.json",
         ),
-        "voc_2007_trainval": ("voc/VOC2007", 'trainval'),
-        "voc_2007_test": ("voc/VOC2007", 'test'),
-        "voc_2012_train": ("voc/VOC2012", 'train'),
-        "voc_2012_trainval": ("voc/VOC2012", 'trainval'),
-        "voc_2012_val": ("voc/VOC2012", 'val'),
-        "voc_2012_test": ("voc/VOC2012", 'test'),
-
+        "voc_2007_train": (
```

Comment:
instead of replacing all the uses that we had before with json-style, what do you think about having a different name tag for COCO-style dataset, like `voc_2007_trainval_cocostyle`?